static/js/issuesTable.js

```diff
@@ -1,35 +1,86 @@
 const IssuesTable = {
-    props: ['engagement', 'engagementsList'],
+    props: ['engagement'],
     components: {
         'board-creation-modal': BoardCreationModal,
         'ticket-creation-modal': TicketCreationModal,
         'filter-toolbar-container': FilterToolbarContainer,
+        'small-issues-table': SmallIssuesTable,
+        'ticket-view-container': TicketViewContainer,
     },
     data() {
         return {
             url: issues_api_url,
             issues_url: issues_api_url,
             noTicketSelected: true,
             preFilterMap: {},
-            table_id: "#issues-table"
+            table_id: "#issues-table",
+            selectedTicket: null,
+            pageNumber: 1,
+            maxPageCount: 1,
+
+            all_users: [],
         }
     },
     mounted(){
         this.setTableCheckEvents()
+        $(this.table_id).on('click-row.bs.table', (e, row, $el, field) => {
+            if (field != "actions"){
+                this.selectedTicket = row
+            }
+        })
+
+        $(this.table_id).on('page-change.bs.table', (e, number) => {
+            this.pageNumber = number
+        })
+
+        $(this.table_id).on('load-success.bs.table', (e, data) => {
+            itemsCount = data.total;
+            this.maxPageCount = Math.ceil(itemsCount/10)
+        })
     },  
     watch: {
+        async selectedTicket(value){
+            if (!value){
+                await this.setUsersOptions()
+            }
+        },
+
         engagement(value){
             notAllEngagements = value.id!=-1
             if (notAllEngagements){
                 this.preFilterMap['engagement'] = value.hash_id
-                $(this.table_id).bootstrapTable('hideColumn', 'engagement')
+                $(this.table_id).bootstrapTable('hideColumn', 'engagement.name')
             } else {
                 delete this.preFilterMap['engagement']
-                $(this.table_id).bootstrapTable('showColumn', 'engagement')
+                $(this.table_id).bootstrapTable('showColumn', 'engagement.name')
             }
         },
     },
     methods: {
+        async setUsersOptions(){
+            generateHtmlOptions = (items, idField='id', titleField='name', currentUserId=null)=>{
+                result = items.reduce((acc, curr) => {
+                    selected = curr[idField] == currentUserId ? "selected" : "" 
+                    return acc + `<option value="${curr[idField]}" ${selected}>${curr[titleField]}</option>`
+                }, '')
+                return result
+            }
+            setOptions = (htmlText, selectId) => {
+                $(selectId).append(htmlText)
+                $(selectId).selectpicker('refresh')
```

Comment:
and here.
+ in majority of cases jquery functions return themselves allowing chaining, e.g.:
`$(selectId).selectpicker('render').selectpicker('refresh')`